semgrep-core/tests/OTHER/rules/relevant_rule_badutf8.js

```diff
@@ -0,0 +1,3 @@
+// x � y : z
+//ruleid: test
+foo

```

Comment:
The title gives me a hint, but just from looking at this test it's not at all obvious to me how this test works. Would be a helpful comment!